src/Analysis/AtlasDataDictionary/AtlasDataDictionaryTable/Components/SearchBar/SearchBar.tsx

```diff
@@ -2,55 +2,35 @@ import React, { useEffect } from 'react';
 import { Input } from '@mantine/core';
 import { debounce } from 'lodash';
 import SearchIcon from '../Icons/SearchIcon';
-import { IRowData } from '../../Interfaces/Interfaces';
+import { IColumnManagementData, IRowData } from '../../Interfaces/Interfaces';
+import SearchBarMessage from './SearchBarMessage/SearchBarMessage';
+import filterTableData from './filterTableData';
 
 interface ISearchBarProps {
   columnsShown: number;
   TableData: IRowData[];
-  setData: Function;
+  paginatedData: IRowData[];
+  setDisplayedData: Function;
+  columnManagementData: IColumnManagementData ;
   searchTerm: string;
   handleTableChange: Function;
+  columnManagementReset: Function
 }
 
 const SearchBar = ({
   columnsShown,
   TableData,
-  setData,
+  paginatedData,
+  setDisplayedData,
+  columnManagementData,
   searchTerm,
   handleTableChange,
+  columnManagementReset,
 }: ISearchBarProps) => {
-  const filterTableData = () => {
```

Comment:
why was this function moved to a separate file?

Reply:
Moved to a separate file to provide information hiding / abstraction and unit testing. Unit tested added for this function now.